Automattic/wp-calypso
# Hosting Command Palette:  Update referrer value on business-related URLs

- Related to https://github.com/Automattic/dotcom-forge/issues/4753

## Proposed Changes

* Use `command-palette` `ref` parameter for register domain, add new site, add jetpack site, and import site
* The `Add Jetpack to a self-hosted site` command didn't change and is still using `cta_from`.

## Testing Instructions


* Access `/sites` and open the command palette with `cmd+k` on Mac or `ctrl+k` on Windows
* Try `Add new site`, `Import site to WordPress.com`, `Register domain` 
* Observe the URLs contain the `ref` URL query parameter.

## Pre-merge Checklist
